fs/accounting/stats.go

```diff
@@ -624,17 +624,28 @@ func (s *StatsInfo) RemoveTransfer(transfer *Transfer) {
 	s.mu.Unlock()
 }
 
-// PruneTransfers makes sure there aren't too many old transfers
-func (s *StatsInfo) PruneTransfers() {
+// pruneTransfers removes transfers that already finished
+//
+// Must be called with the lock held
+func (s *StatsInfo) pruneTransfers() {
```

Comment:
Usually on such occasions it's called `pruneTransfersLocked`, we could move it below the main function too.

Reply:
I followed `RemoveTransfer` convention. I'll leave it as it is.